app/airq/models/clients.py

```diff
@@ -301,3 +305,21 @@ def maybe_notify(self) -> bool:
 
     def log_event(self, event_type: EventType, **event_data: typing.Any) -> Event:
         return Event.query.create(self.id, event_type, **event_data)
+
+    def get_last_client_event(self) -> typing.Optional[Event]:
```

Comment:
Did you decide not to rename this to `get_last_inbound_event`?

Reply:
Yeah I did, because I think the word client is clear enough since we are in the client model